spec/models/category_spec.rb

```diff
@@ -1,5 +1,21 @@
 require 'spec_helper'
 
 describe Category do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { should have_many(:videos)}
+
+  #it "save itself" do
```

Comment:
Make sure you clean up commented specs too. If it is commented, it is not needed.